ReactAndroid/src/main/java/com/facebook/react/views/art/ARTShapeShadowNode.java

```diff
@@ -182,6 +234,21 @@ protected boolean setupFillPaint(Paint paint, float opacity) {
               (int) (mFillColor[2] * 255),
               (int) (mFillColor[3] * 255));
           break;
+        case 1:
+          int stopsCount = getStopsCount(mFillColor);
+          int [] stopsColors = new int [stopsCount];
+          float [] stops = new float[stopsCount];
+          gradientStopsParsing(mFillColor, stopsCount, stops, stopsColors);
+          paint.setShader(
+            new LinearGradient(
+              mFillColor[3] * 2,
```

Comment:
why the order is `3 4 1 2`?
```
mFillColor[1] * 2,
mFillColor[2] * 2,
mFillColor[3] * 2,
mFillColor[4] * 2,
```
This will draw the same result as iOS did